tests/Kernel/fixtures/rendering.yml

```diff
@@ -162,8 +162,8 @@
       'div.ecl-hero-banner__image[style="background-image:url(http://via.placeholder.com/150x150)"]': 1
       'div.ecl-page-banner--full-width': 0
     equals:
-      'div.ecl-container.ecl-hero-banner__container div.ecl-hero-banner__content h1.ecl-hero-banner__title': Banner title
-      'div.ecl-container.ecl-hero-banner__container div.ecl-hero-banner__content p.ecl-hero-banner__description': Banner description
+      'div.ecl-container div.ecl-hero-banner__container div.ecl-hero-banner__content div.ecl-hero-banner__title': Banner title
```

Comment:
There are new banners variants like with or without text block and with white background that should be tested.
<img width="180" alt="banners variants" src="https://user-images.githubusercontent.com/22908988/92571698-ba646280-f283-11ea-84a1-3c0dea1d9e96.png">

Reply:
Created a follow up: https://citnet.tech.ec.europa.eu/CITnet/jira/browse/EWPP-235